src/flex/FlexView.js

```diff
@@ -76,6 +73,17 @@ const PropTypes = {
 @props(PropTypes, { strict: false })
 export default class FlexView extends React.Component {
 
+  componentDidMount() {
+    this.logWarnings();
+  }
+
+  logWarnings = () => {
+    const { auto } = this.props;
+    if (!t.Nil.is(auto)) {
+      warn('"auto" is DEPRECATED. Use combination of "grow", "shrink" and "basis" instead');
```

Comment:
semver + breaking change changelog :+1: 

provided someone is will to fix this in all our projects at once :P